analysis_options.yaml

```diff
@@ -1 +0,0 @@
-include: package:pedantic/analysis_options.yaml
```

Comment:
You have to include the analysis_options from Flutter lints
Like this:
include: package:flutter_lints/flutter.yaml

Reply:
Resolved